### PR TITLE
Completed assignment

### PR DESCRIPTION
**Pull request by dabellator**
I tried to include all the necessary files (readme, license, gulp etc) please let me know if i forgot anything.  thanks!  btw, do you have a good resource for formatting a markdown file?

### CODE REVIEW

File: jordan_bundy/LICENSE [hunk 1]
```diff
@@ -0,0 +1,25 @@
+opyright (c) <year> <copyright holders>
```
Comment (mollyfish) [inline jordan_bundy/LICENSE:1]: Make sure you put in the year and your name (or your team)!  Also the "c" is missing from "copyright"

File: jordan_bundy/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+MY INSTRUCTIONS:
+</br>
+launch `node server.js`
+</br>
+send a post request to localhost:3000/data
+</br>
+check the file at data/input.json, it will include your input
+
```
Comment (mollyfish) [inline jordan_bundy/README.md:9]: It wont let me comment below - your .keep should actually be named .gitkeep (it's a little more informative about why you have this empty, hidden file in the directory)

File: jordan_bundy/LICENSE (hunk 1 above)
Comment (dabellator) [inline jordan_bundy/LICENSE:1]: Thanks!

File: jordan_bundy/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
```
Comment (mollyfish) [inline jordan_bundy/README.md:1]: GitHub has a nice markdown 'cheatsheet,' here: https://guides.github.com/features/mastering-markdown/

File: jordan_bundy/server.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+app.get('/data', function(req, res) {
+  var readStream = fs.createReadStream(__dirname + '/data/input.json');
+  res.writeHead(200, {'content-type':'application/json'});
+  readStream.pipe(res);
+});
+
+app.post('/data', function(req, res) {
+  var writeStream = fs.createWriteStream(__dirname + '/data/input.json');
+
```
Comment (mollyfish) [inline jordan_bundy/server.js:13]: So, this code will allow me to save a single post to a file called `input.json` - but the goal is to be able to make multiple posts, named whatever I want (file names based on the url I post to).  For example, if I post to `/data/molly_note`, it should make `molly_note.json` without overwriting any other files.  Then I can make a get request to `/data/molly_note` and it should spit back the file contents.

You are close!  Just add the filename capability and you should get full credit.

Comment (dabellator) [inline jordan_bundy/server.js:13]: Thanks Molly, I must have misunderstood Tyler.  I was working on that, and heard him say don't worry about updating the file.  I like the idea of being able to update the resource though, so I'll go back and complete this after this week.

**Pull request merged**